pom.xml

```diff
@@ -157,7 +157,47 @@
         <module>stormbench</module>
       </modules>
       <activation>
-        <activeByDefault>true</activeByDefault>
+        <property>
+          <name>!bench</name>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>defaulthadoop</id>
+      <properties>
+        <hadoop.mr2.version>3.2.1</hadoop.mr2.version>
+      </properties>
+      <activation>
+        <property>
+          <name>!hadoop</name>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>haoop2.4</id>
+      <properties>
+        <hadoop.mr2.version>2.4.0</hadoop.mr2.version>
+      </properties>
+      <activation>
+        <property>
+          <name>hadoop</name>
+          <value>2.4</value>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>haoop3.2</id>
+      <properties>
+        <hadoop.mr2.version>3.2.1</hadoop.mr2.version>
+      </properties>
+      <activation>
+        <property>
+          <name>hadoop</name>
+          <value>3.2</value>
+        </property>
```

Comment:
Could we also add hadoop 3.1, 3.0 profile tags?

Reply:
@gczsjdy  it's done.

Reply:
Like that!

Reply:
I suggest that in Travis CI we also add a hadoop 3.1/3.0 version build case.

Reply:
@gczsjdy  I have increased hadoop 3.1/3.0 version in Travis CI. Please , double check.